src/commands/github-bot-comment-on-pr.yml

```diff
@@ -37,6 +42,23 @@ steps:
         echo "## << parameters.title >>" > /tmp/preamble.txt
         cat /tmp/preamble.txt /tmp/notification.txt > /tmp/constructed_forslack.txt
 
+        if [ $(du -k /tmp/constructed_forslack.txt | cut -f1) -gt 65 ]; then
+           # get file size in K. 65K is too big for Github Pull Requests Comments (API errors)
+           if [ "<< parameters.overlarge-content-to >>" = "report.md" ]; then
+               cp /tmp/constructed_forslack.txt /tmp/report.md
+
+               echo "## << parameters.title >>" > /tmp/constructed_forslack.txt
+               echo "Output redirected to the full report, linked below, as otherwise comment would be too long." >> /tmp/constructed_forslack.txt
+            fi
+
+            if [ "<< parameters.overlarge-content-to >>" = "print" ]; then
```

Comment:
Is it possible to break the output into multiple comments instead (or as a third option)? This change does fix the silent failure issue but doesn't really address the "output isn't easily available" usability aspect.

Reply:
My gut says to leave that to the caller - if there's some logical grouping of things inside this very large comment the caller may know better than us.

Maybe a future feature may be to support `/tmp/notifications_1.txt` `/tmp/notifications_2.txt` etc etc, or give the user a better ability to loop through whatever grouping they defined upstream. (But ideally I don't want to implement that in this PR)

Reply:
Sounds reasonable, it's definitely scope creep. But I will propose that we leave our internal ticket open - a message that says "go look at Circle" isn't _really_ fixing the "output doesn't display in my PR" issue 😆